pages/icon/icons-list.tsx

```diff
@@ -9,15 +9,15 @@ import ScreenshotArea from '../utils/screenshot-area';
 
 import styles from './icons-list.scss';
 
-const sizes = ['small', 'normal', 'big', 'large'] as const;
+const sizes = ['small', 'normal', 'medium', 'big', 'large'] as const;
 
 export default function IconsList({ variant }: { variant: IconProps['variant'] }) {
   const className = variant === 'inverted' ? styles.invertedIconsScenario : undefined;
   return (
     <ScreenshotArea key={variant} className={className}>
       <h1 className={styles.header}>{variant}</h1>
       {sizes.map(size => (
-        <div key={size}>
+        <div key={size} style={{ paddingBlockEnd: 8, paddingBlockStart: 2, display: 'flex', flexWrap: 'wrap', gap: 4 }}>
```

Comment:
It seems that we already have a `icons-list.scss` file next to this one, I think it would be better if we move styles there.

Reply:
That makes sense. I moved the styles to the style sheet.